### ISSUE TITLE
Multiple dex files define Lcom/google/android/gms/iid/zzc;

### ISSUE THREAD

**Issue by ps84** [transistorsoft/cordova-background-geolocation-lt#386]
I built a proper cordova environment and now I'm getting the Dex error when I try to build.  I've narrowed down the conflict between your plugin and phonegap-push - both are critical to the success of our app.

This exact problem is described at https://github.com/phonegap/phonegap-plugin-push/blob/c350ce2c0cffb763eb2b1a84295cd701bf33693a/docs/INSTALLATION.md#multidex which suggests talking to you (their words).

Thanks again

## Your Environment
* Plugin version: 2.7.1
* Platform: Android 
* OS version: Windows 7
* Device manufacturer / model:
* Cordova version (`cordova -v`):  7.0.1
* Cordova platform version (`cordova platform ls`): 5.2
* Plugin config

c:\projects\cordova\bggeo>cordova plugin list
cordova-background-geolocation-lt 2.7.3 "BackgroundGeolocation"
cordova-plugin-background-fetch 4.0.0 "CDVBackgroundFetch"
cordova-plugin-dialogs 1.3.3 "Notification"
cordova-plugin-whitelist 1.3.2 "Whitelist"
phonegap-plugin-push 1.8.4 "PushPlugin"

## Actual Behavior

* What went wrong:
Execution failed for task ':transformClassesWithDexForDebug'.
> com.android.build.api.transform.TransformException: com.android.ide.common.process.ProcessException: java.util.concurrent.ExecutionException: com.android.dex.DexException: Multiple dex files define
Lcom/google/android/gms/iid/zzc;

## Steps to Reproduce

1.
2.
3.
4.

**Comment by christocracy** [comment]
The problem is `phonegap-plugin-push` is pinning play-services in the version you're using (1.8.4) at [9.8+](https://github.com/phonegap/phonegap-plugin-push/blob/v1.8.4/plugin.xml#L61) while my plugin requests `:+` (latest version, `11.0.1`).

Upgrade to latest `phonegap-plugin-push` (1.10.5)
![](https://www.dropbox.com/s/xfq4i2ijgpk7bbj/Screenshot%202017-06-27%2012.03.50.png?dl=0)

**Comment by ps84** [comment]
I suspected that was the case but, as I've mentioned via e-mail, I'm stuck on Appery for now and they're stuck on Cordova 5.  I had read about a few similar cases that were solved by tweaking gradle and had hoped this might be the case here but I understand that this is complicated stuff.  In the meantime I've downgraded to v1.4.5 which still appears to work for me and will hopefully carry me until I can build a proper development environment.

Thanks again, really appreciate it.
